DependencyInjection/GrossumFaqExtension.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Grossum\FaqBundle\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+class GrossumFaqExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config        = $this->processConfiguration($configuration, $configs);
```

Comment:
@aivus ok i agree with you. I think we should export configs for phpStorm and add them to wiki. Everybody who work in symfony project must use it.  
@cowboy4ik89 please, reformat this code.